src/brs/http/JSONData.java

```diff
@@ -1,22 +1,129 @@
 package brs.http;
 
-import brs.*;
+import static brs.http.common.ResultFields.ACCOUNT_RESPONSE;
```

Comment:
I would revert this bit back to `import brs.*`. I assume this was an IDE change, thinking its helping.

Reply:
Yes, it was an automatic IDE change. Sorry for that. I will revert it.